### PR TITLE
feat: structure folder and added callback session on next-auth

### PR DESCRIPTION
**Pull request by muhammadanova**

### CODE REVIEW

File: src/pages/api/auth/[...nextauth].ts [hunk 1]
```diff
@@ -1,15 +1,30 @@
...
 import GitHubProvider from "next-auth/providers/github";
 import prisma from "@/lib/prisma";
-
 import type { NextApiHandler } from "next";
 
+declare module "next-auth" {
+  interface Session extends DefaultSession {
+    user: {
+      id: string;
+    } & DefaultSession["user"];
```
Comment (dannycahyo) [inline src/pages/api/auth/[...nextauth].ts:11]: [ASK], why the typing can't be directly coming from the `DefaultSession` ya mas?

File: src/pages/index.tsx [hunk 2]
```diff
@@ -1,6 +1,8 @@
 import Head from "next/head";
 import { Inter } from "next/font/google";
-import styles from "@src/styles/Home.module.css";
+import styles from "@/styles/Home.module.css";
```
Comment (dannycahyo) [inline src/pages/index.tsx:3]: Since the Home module css has been deleted, I don't think we need to import this anymore mas.

Comment (muhammadanova) [reply src/pages/index.tsx:3]: I removed duplicated folder pages and style. So home modules keep needed on tag main Dan

Comment (dannycahyo) [reply src/pages/index.tsx:3]: oh I see, I just realized that you deleted the root styles directory.

**Review by dannycahyo — APPROVED** [review]

**Pull request merged**